CMakeLists.txt

```diff
@@ -68,8 +68,7 @@ endif()
 ADD_CUSTOM_TARGET(uninstall
   "${CMAKE_COMMAND}" -P "${CMAKE_CURRENT_BINARY_DIR}/cmake_uninstall.cmake")
 
-# identify target architecture, build portaudio before any SC specific
-# definitions break the pa-build
```

Comment:
this is a noteworthy comment. you don't need to change anything, but we may end up needing to remove more flags as now everything below can potentially be defined when compiling portaudio. that one define is probably the only one. after all we've been building boost libs and yamlcpp this way for quite a while with no issues.

Reply:
Do you mean that we've been building everything with flags set by PortAudio and now we won't?

Reply:
i mean that now we are building with all the flags set later on in this file (`-DBOOST_DATE_TIME_NO_LIB -DBOOST_ALL_NO_LIB` etc.), where before we weren't